### ISSUE TITLE
[Feature] EIP-7691

### ISSUE THREAD

**Issue by onbjerg** [alloy-rs/alloy#1712]
### Component

consensus, eips, genesis

### Describe the feature you would like

EIP-7691 is mostly a CL change, however, it will also change `BLOB_BASE_FEE_UPDATE_FRACTION` in the EL to change the fee market a bit. This is a patchwork solution until we gather more info about blob fee markets for Fusaka.

See the relevant PR that adds these EL changes: https://github.com/ethereum/EIPs/pull/9060/files

TL;DR: For < Prague we use the current `BLOB_BASE_FEE_UPDATE_FRACTION`, for >= Prague use `BLOB_BASE_FEE_UPDATE_FRACTION_ELECTRA`.

Relevant snippets: https://github.com/alloy-rs/alloy/blob/20d13faff78325dda42de8952e0addad31d11db7/crates/eips/src/eip4844/mod.rs#L70-L71

https://github.com/alloy-rs/alloy/blob/20d13faff78325dda42de8952e0addad31d11db7/crates/eips/src/eip4844/mod.rs#L131-L142

**Comment by nadtech-hub** [comment]
:hand:
